cocos/ui/view.ts

```diff
@@ -596,8 +595,10 @@ export class View extends Eventify(System) {
         const w = this._designResolutionSize.width;
         const h = this._designResolutionSize.height;
 
-        if (width > 0) {
+        if (w > 0 && width > 0 && h > 0 && height > 0) {
```

Comment:
Why should check design resolution size here? I think its value should be checked in setting its value.

Reply:
The older code only check width maybe a mistake. It may be produced by this [pr](https://github.com/cocos/cocos-engine/commit/e7e8036dcd78d2eec1c53ffba3eed5f8ded8fe69)